src/Room.js

```diff
@@ -319,6 +320,18 @@ function notificationCallback (data) {
     return this.kuzzle.emitEvent('jwtTokenExpired');
   }
 
+  if (data.controller === 'document') {
+    data.type = 'document';
+    data.document = new Document(this.collection, data.result._id, data.result._source);
+    delete data.result;
+  }
+
+  if (data.controller === 'realtime') {
+    data.type = 'user';
+    data.user = {count: data.result.count};
```

Comment:
Same question for me, but I did not follow the discussion about that, so I may be wrong... 
Can you either fix that, or fix the PR description, depending what is really wanted ?